data/content_scripts/posting_process/sendkeys.js

```diff
@@ -0,0 +1,766 @@
+// Cross-broswer implementation of text ranges and selections
+// documentation: http://bililite.com/blog/2011/01/17/cross-browser-text-ranges-and-selections/
+// Version: 2.6
+// Copyright (c) 2013 Daniel Wachsstock
+// MIT license:
```

Comment:
Yeah, that's exactly what I do [here](https://github.com/privly/privly-jetpack/pull/5/files#diff-72a8aee6ddb986cc37b045a74f2f90baR220), i.e, once we have the privly url, attach a content script that posts the url, and then [remove](https://github.com/privly/privly-jetpack/pull/5/files#diff-72a8aee6ddb986cc37b045a74f2f90baR231) the script once the posting process is over. But this won't work when dealing with iframes, I can't inject a content script into a specific iframe(unless I attach a content script on every page load).